docs/Primer/Containers-and-Casting.md

```diff
@@ -0,0 +1,251 @@
+## Lists
+
+!!! note
+    **List** - A linked list that can hold a variable amount of objects.
```

Comment:
Neither `Boo.Lang.List` nor `System.Collections.Generic.List<>` is implemented as a *linked list*.